### DISCUSSION TITLE
Mendukung Interactive Message

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#143] [Roadmap]
> Update 23 Feb 2023: Fitur sudah diluncurkan lihat #167 

Interactive message berguna untuk memandu pengguna untuk memilih atau mengirimkan pesan sesuai dengan apa yang kita inginkan. Fitur yang rencana didukung adalah:

- List Message
- Reply Button

Referensi lengkap tentang interactive dapat dilihat di [dokumentasi resmi WhatsApp](https://developers.facebook.com/docs/whatsapp/guides/interactive-messages/).

**Comment by zonegoh** [comment]
mas rio, secepatnya direalisasikan yah :D

**Reply by rioastamal** [reply]
Siap mas @zonegoh , terima kasih dukungannya.

**Reply by rioastamal** [reply]
@zonegoh fitur sudah diluncurkan lihat #167

**Comment by dp20bp** [comment]
fitur "Reply Button" di KirimWA.id saat ini apakah sudah bisa mas @rioastamal?

**Reply by rioastamal** [reply]
Sudah, ada di #167
